lib/commands-queue.ts

```diff
@@ -44,13 +44,13 @@ export default class RedisCommandsQueue {
     static encodeCommand(args: Array<string>): string {
         const encoded = [
             `*${args.length}`,
-            `$${args[0].length}`,
+            `$${Buffer.byteLength(args[0])}`,
             args[0]
         ];
 
         for (let i = 1; i < args.length; i++) {
             const str = args[i].toString();
```

Comment:
```suggestion
```
As `args` is `string[]`, this line also seems to need to be reviewed. Is it casting `string` to a `string`?